.github/workflows/publish.yaml

```diff
@@ -0,0 +1,490 @@
+# Based on https://github.com/prettier/prettier/blob/master/.github/workflows/dev-test.yml
+name: Publish to PyPi
+
+on:
+  workflow_dispatch:
+    inputs:
+      randomlyseed:
+        description: 'Pytest Randomly Seed'
+        required: false
+        default: ''
+      publish_to_pypi:
+        description: 'Publish to PyPi'
+        required: false
+        default: '0'
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - "ubuntu-18.04"
+          - "ubuntu-20.04"
+          - "windows-2019"
+          - "macos-10.15"
+          - "macos-11.0"
+        python-version:
+            - 3.7
+            - 3.8
+            - 3.9
+        include:
+          # only enable coverage on the fastest job
+          - os: "ubuntu-18.04"
+            python-version: "3.7"
```

Comment:
update fastest job to latest version of linux (ubuntu 20) and latest version of python (3.9). i've been meaning to do that in the template